src/test/java/de/tum/in/www1/artemis/service/FileServiceTest.java

```diff
@@ -394,4 +396,54 @@ void testIgnoreDirectoryFalsePositives(@TempDir Path targetDir) throws IOExcepti
         final Path expectedTargetFile = targetDir.resolve("jenkins").resolve("package.xcworkspace");
         assertThat(expectedTargetFile).doesNotExist();
     }
+
+    @Test
+    void testSanitizeByCheckingIfPathContainsSubPathElseThrow_Background_Ignore_Path_Null() {
+        assertThatCode(() -> FileService.sanitizeByCheckingIfPathContainsSubPathElseThrow(null, URI.create("/api/" + FileService.DRAG_AND_DROP_BACKGROUND_SUBPATH + "/")))
+                .doesNotThrowAnyException();
+    }
+
+    @Test
+    void testSanitizeByCheckingIfPathContainsSubPathElseThrow_Background_Ignore_SubPath_Null() {
+        assertThatCode(() -> FileService.sanitizeByCheckingIfPathContainsSubPathElseThrow(URI.create("/api/files/drag-and-drop/backgrounds/1/../../BackgroundFile.jpg"), null))
+                .doesNotThrowAnyException();
+    }
+
+    @Test
+    void testSanitizeByCheckingIfPathContainsSubPathElseThrow_Background_Ignore_Path_Temp() {
```

Comment:
I would specifically expect that `/api/files/temp/backgrounds/1/../../../exam-users/signatures/some-file.png` would throw an exception.

Reply:
Case included in [57ebbd70da44d589f7c61fa9f42a7d4742ab1d71](https://github.com/ls1intum/Artemis/pull/7170/commits/57ebbd70da44d589f7c61fa9f42a7d4742ab1d71).